### DISCUSSION TITLE
Approved on the 25th but I still can't access

### DISCUSSION THREAD

**Discussion by keichigo** [community/community#65973] [GitHub Education] [answered]
### Hi everyone,

I still don't have access to the benefits, I thought it took around 6 days or less, is this normal?

### A suggestion...(optional)

 on the 25th

**Comment by mehulrajdev** [comment, marked as answer]
You will get

**Comment by nirav-gajera** [comment]
It is normal for it to take a few days for your benefits to be activated after you are approved. However, if it has been more than 6 days, you can contact [GitHub support](https://support.github.com) to check on the status of your application.

**Comment by keichigo** [comment]
ok
